weather_aplication/SchowWeatherViewController.swift

```diff
@@ -0,0 +1,118 @@
+import SnapKit
+import UIKit
+
+struct CellData {
```

Comment:
Tutaj też by się lepsza nazwa przydała .CellData w tym momencie odnosi się do tego gdzie to ma być użyte, a powinno odzwierciedlać pola w nim zawarte